filebeat/harvester/log.go

```diff
@@ -286,6 +285,15 @@ func (h *Harvester) close() {
 	// If file was never opened, it can't be closed
 	if h.file != nil {
 
+		if h.config.Compression == config.GZipCompression {
```

Comment:
In all other cases, offset == size of the file in case the harvester finished reading. For gzip files the offset is > Size(). But to not have to change the prospector logic, it is set to the size of the file. It is kind of a hack, means I should add a doc comment here.